src/mocks/repositories/locationRepository.ts

```diff
@@ -65,6 +71,39 @@ class LocationRepository implements Repository<Location> {
     return newItem;
   }
 
+  store(store: StoreInput[] | StoreInput, locationBarcode: string): Location {
```

Comment:
Can you explain what this function does please?

Reply:
This is actually a mockup repository to replicate the Storelight storage. This method is to mockup what 'store' api does.  It stores all given barcodes in the given location in a mockup repository.  It goes through all stored items in the repository and checks if any of the given barcodes is stored elsewhere. If yes, it removes that barcode from old location. Then it is added in new location.